packages/default/scss/expansion-panel/_layout.scss

```diff
@@ -0,0 +1,59 @@
+@include exports("expansion-panel/layout") {
+
+    // Base
+    .k-expander {
+        @include use-background-clip();
+
+        border-width: $expander-border-width;
+        border-style: solid;
+        box-sizing: border-box;
+        outline: 0;
+        font-family: $expander-font-family;
+        font-size: $expander-font-size;
+        line-height: $expander-line-height;
+        display: flex;
+        flex-direction: column;
+        -webkit-touch-callout: none;
+        -webkit-tap-highlight-color: $rgba-transparent;
+
+        // Header
+        .k-expander-header {
+            padding: 0 $expander-header-padding-x $expander-header-padding-y;
+            border-width: 0 0 $expander-header-border-width;
+            border-style: solid;
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            cursor: pointer;
+        }
+
+        // Expander spacer
+        .k-expander-spacer {
+            flex: 1 1 auto;
+        }
+
+        // Title
+        .k-expander-title {
+            text-transform: uppercase;
+        }
+
+        // Expand / collapse icon
+        .k-expander-indicator {
+            margin-left: $expander-indicator-margin-x;
+        }
+
+        // Expander content
+        .k-expander-content {
+            padding: $expander-content-padding-y $expander-content-padding-x;
+            overflow: auto;
+        }
+
+        // Multiple expansion panels immediately one after another
+        +.k-expander {
+            .k-expander-header {
+                padding-top: $expander-padding-y;
+            }
+        }
```

Comment:
```suggestion
        + .k-expander .k-expander-header {
            padding-top: $expander-padding-y;
        }
```

Reply:
We usually use nesting for the selectors. What's the reason for this change

Reply:
Indeed, we are using nested selectors for easier implementation. However, nesting a selector slowers the build and it is harder to read/find in code when only a single declaration is used.

We can add this as a question to the agenda of our tomorrow's meeting, and discuss this topic with other FEs. After that, we can add our decision to the following wiki - https://github.com/telerik/kendo-themes/wiki/Code-Styleguide